icu4c/source/i18n/listformatter.cpp

```diff
@@ -179,6 +178,42 @@ const ListFormatInternal* ListFormatter::getListFormatInternal(
     return result;
 }
 
+static const char* typeWidthToStyleString(UListFormatType type, UListFormatWidth width) {
+    switch (type) {
+        case ULISTFMT_TYPE_CONJUNCTION:
+            switch (width) {
+                case ULISTFMT_WIDTH_SHORT:
+                    return "standard";
+                default:
```

Comment:
Please add  case for ULISTFMT_WIDTH_SHORT: nor ULISTFMT_WIDTH_NARROW: here

Reply:
CONJUNCTION and DISJUNCTION only support SHORT width.  Why do I need to add cases for the others?  It will just cause a U_ILLEGAL_ARGUMENT_ERROR.

Reply:
Disagree. Read https://unicode.org/reports/tr35/tr35-general.html#ListPatterns
Notice "type attribute value" include all 9 combinations
Also see https://github.com/unicode-org/icu/blob/master/icu4c/source/data/locales/en.txt#L2007